commands/ls.go

```diff
@@ -87,6 +87,9 @@ func parseFilters(filters []string) (FilterOptions, error) {
 	options := FilterOptions{}
 	for _, f := range filters {
 		kv := strings.SplitN(f, "=", 2)
+		if len(kv) != 2 {
+			return options, fmt.Errorf("Unupported filter syntax, please use help ls for details on correct usage.")
```

Comment:
You can probably just shorten this to "Unsupported filter syntax".  Consulting the help text is largely implied, I feel.